tests/CheckTranslationsCommandTest.php

```diff
@@ -0,0 +1,56 @@
+<?php
+
+namespace NextApps\PoeditorSync\Tests;
+
+use Illuminate\Filesystem\Filesystem;
+use NextApps\PoeditorSync\Poeditor\Poeditor;
+
+class CheckTranslationsCommandTest extends TestCase
+{
+    protected function setUp() : void
+    {
+        parent::setUp();
+
+        app(Filesystem::class)->cleanDirectory(lang_path());
+        app(Filesystem::class)->makeDirectory(lang_path('en'));
+    }
+
+    /** @test */
+    public function it_checks_if_local_translations_match_downloaded_translations()
+    {
+        config()->set('poeditor-sync.locales', ['en', 'nl']);
+
+        $this->createPhpTranslationFile('en/php-file.php', ['foo' => 'bar', 'nested' => ['value' => 'nested value'], 'baz' => 'bar']);
+
+        $this->mockPoeditorDownload('en', [
+            'php-file' => [
+                'foo' => 'bar',
+                'nested' => [
+                    'value' => 'nested value',
+                ],
+            ],
+        ]);
+
+        $this->mockPoeditorDownload('nl', [
+            'php-file' => [
+                'bar_foo' => 'foo bar',
+            ],
+        ]);
+
+        $this->artisan('poeditor:check')
```

Comment:
can you extend the test to fix every language one by one and check that the output of the command changes (also use "doesntExpectOutput")